app/src/organisms/ODD/QuickTransferFlow/QuickTransferAdvancedSettings/PreWetTip.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react'
+import { createPortal } from 'react-dom'
+import { useTranslation } from 'react-i18next'
+
+import {
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  POSITION_FIXED,
+  RadioButton,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import { getTopPortalEl } from '/app/App/portal'
+import { i18n } from '/app/i18n'
+import { ChildNavigation } from '/app/organisms/ODD/ChildNavigation'
+import { useTrackEventWithRobotSerial } from '/app/redux-resources/analytics'
+import { ANALYTICS_QUICK_TRANSFER_SETTING_SAVED } from '/app/redux/analytics'
+
+import { ACTIONS } from '../constants'
+
+import type { Dispatch } from 'react'
+import type {
+  FlowRateKind,
+  QuickTransferSummaryAction,
+  QuickTransferSummaryState,
+} from '../types'
+
+interface PreWetTipProps {
+  onBack: () => void
+  state: QuickTransferSummaryState
+  dispatch: Dispatch<QuickTransferSummaryAction>
+  kind: FlowRateKind
+}
+
+export function PreWetTip(props: PreWetTipProps): JSX.Element {
+  const { kind, onBack, state, dispatch } = props
+  const { t } = useTranslation('quick_transfer')
+  const { trackEventWithRobotSerial } = useTrackEventWithRobotSerial()
+  const [preWetTipIsEnabled, setPreWetTipIsEnabled] = useState<boolean>(
+    state.preWetTip
+  )
+  const enablePreWetTipDisplayItems = [
+    {
+      option: true,
+      description: t('option_enabled'),
+      onClick: () => {
+        setPreWetTipIsEnabled(true)
+      },
+    },
+    {
+      option: false,
+      description: t('option_disabled'),
+      onClick: () => {
+        setPreWetTipIsEnabled(false)
+      },
+    },
+  ]
+
+  const handleClickBackOrExit = (): void => {
+    onBack()
+  }
+
+  const handleClickSave = (): void => {
+    dispatch({
+      type: ACTIONS.SET_PRE_WET_TIP,
+      preWetTip: !state.preWetTip,
+    })
+    trackEventWithRobotSerial({
+      name: ANALYTICS_QUICK_TRANSFER_SETTING_SAVED,
+      properties: {
+        setting: `Pre-wet-tip_${kind}`,
+      },
+    })
+    onBack()
+  }
+
+  const setSaveOrContinueButtonText = t('shared:save')
```

Comment:
nit, since copy always = `save`? 
```suggestion
  const setSaveButtonText = t('shared:save')
```